phoenix-assembly/src/build/components/all-common-jars.xml

```diff
@@ -55,13 +47,5 @@
         <include>phoenix-pherf.jar</include>
       </includes>
     </fileSet>
-    <fileSet>
-      <directory>${project.basedir}/../phoenix-tools/target</directory>
```

Comment:
I'm curious -- why does this affect phoenix-tools?

Reply:
We've dropped the phoenix-tools module/artifact some time ago, this is just an (unrelated) cleanup.